MapboxCoreNavigation/Feedback.swift

```diff
@@ -26,22 +26,85 @@ public enum FeedbackType: CustomStringConvertible {
 
     /// Indicates a problem with positioning the user
     case positioning(subtype: PositioningSubtype?)
-    
+
+    /*
+      case streetPermanentlyBlockedOff
+      case roadMissingFromMap
+      case other
```

Comment:
What’s this comment for?

Reply:
I'll remove it. I was dropping in the enum as a reference as I built the case statement